gp-includes/things/translation.php

```diff
@@ -661,6 +661,23 @@ public function last_modified( $translation_set ) {
 		return $last_modified;
 	}
 
+	/**
+	 * Get IDs of translations by ID of their original and translation set.
+	 *
+	 * @since 2.3.0
+	 *
+	 * @param int $original_id        ID of original.
+	 * @param int $translation_set_id ID of translation set.
+	 * @return array $ids An array of IDs of matching translations.
+	 */
+	public function ids_by_original_and_set( $original_id, $translation_set_id  ) {
+		global $wpdb;
+
```

Comment:
By preserving existing behaviour, do you mean that by default we should always display link?